proto/api/api.proto

```diff
@@ -280,6 +280,7 @@ message ReadStateRequest {
   bytes protocolID = 1;
   bytes methodName = 2;
   repeated bytes arguments = 3;
+  string height = 4;
```

Comment:
plz add comment saying this field is optional, if not present, read from tip height